sdk/jdbc/azure-identity-providers-jdbc/src/main/java/com/azure/identity/providers/jdbc/implementation/token/AccessTokenResolverOptions.java

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.identity.providers.jdbc.implementation.token;
+
+import com.azure.identity.providers.jdbc.enums.AuthProperty;
+
+import java.util.Properties;
+
+
+/**
+ * Contains details of a request to get a token.
+ */
+public class AccessTokenResolverOptions {
+
+    private String claims;
+    private String tenantId;
+    private String[] scopes;
+    private boolean cacheAccessToken;
+
+    public AccessTokenResolverOptions() {
+
+    }
+
+    public AccessTokenResolverOptions(Properties properties) {
```

Comment:
how about we take a `com.azure.core.util.Configuration` instead of `Properties`?

Reply:
what do you mean mixed together?